### DISCUSSION TITLE
Next Gen Images

### DISCUSSION THREAD

**Discussion by Tigerlady** [MasaCMS/MasaCMS#301] [Ideas]
I would like to see added next-gen file types like webp and avif added to the ability to upload images for pages and to upload these types in the editor.

**Comment by Tigerlady** [comment]
So I have done more research on this and these are the files that would need to be updated. 

/core/mura/content/file/fileDAO.cfc
/core/mura/content/file/fileManager.cfc
/core/mura/content/file/processImgCfimage.cfc
/core/mura/content/contentFileMetaDataBean.cfc
/core/mura/content/contentGatewayAdobe.cfc
/core/mura/content/contentHTMLExporter.cfc
/core/mura/content/contentManager.cfc
/core/mura/content/contentRenderer.cfc
/admin/core/views/carch/assocfiles.cfm
/admin/core/views/carch/dsp_nest.cfm
/admin/core/views/carch/loadsiteflat.cfm
/admin/core/views/carch/multifileupload.cfm
/admin/core/views/carch/selectmedia.cfm
/admin/core/views/csettings/editsite.cfm
/core/mura/configBean.cfc
/core/modules/v1/filebrowser/model/beans/filebrowser.cfc
/core/mura/client/api/json/v1/apiUtility.cfc
/core/mura/client/api/json/v1/jsonApiUtility.cfc

I don't have the line numbers but if you want I can provide those as well.  This would be very helpful with keeping up with the trends of using next-gen imaging.

**Comment by mikechy** [comment]
Second, Third and Fourth this!  We are modifying core files to make this work!
